proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -112,18 +112,23 @@ message Validator {
       [(gogoproto.nullable) = false, (gogoproto.stdtime) = true, (gogoproto.moretags) = "yaml:\"unbonding_time\""];
   // commission defines the commission parameters.
   Commission commission = 10 [(gogoproto.nullable) = false];
-  // min_self_delegation is the validator's self declared minimum self delegation.
-  string min_self_delegation = 11 [
```

Comment:
I do not think this should be deleted but instead deprecated.

Reply:
correct

Reply:
Works for me. Does this just consist of adding back the field to the proto or do you want references to it to be added back as well?

Reply:
Lmk if more needs to be done besides [this](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/97efdacc95f41e7b44b37ea0d526b86ee3a44536)